rkt/rkt.go

```diff
@@ -27,6 +27,7 @@ import (
 	"github.com/coreos/rkt/rkt/config"
 	rktflag "github.com/coreos/rkt/rkt/flag"
 	"github.com/spf13/cobra"
+	_ "github.com/spf13/cobra/doc" // transitive dependency for man page generation
```

Comment:
I think it is ok as long as the comment is here to explain.

It reminds me of this dummy package in old rkt versions:
https://github.com/coreos/rkt/blob/v0.11.0/stage1/dummy.go#L15
which was removed in favor of the file `vendoredApps`.